docs/Reference/API-Objects.md

```diff
@@ -32,6 +32,7 @@ Returned by [`eth_getBlockByHash`](API-Methods.md#eth_getblockbyhash) and
 | **timestamp**        | Quantity              | Unix timestamp for block assembly.                                 |
 | **transactions**     | Array                 | Array of [transaction objects](#transaction-object), or 32 byte transaction hashes depending on the specified boolean parameter. |
 | **uncles**           | Array                 | Array of uncle hashes.                                             |
+| **baseFeePerGas**    | Quantity              | The block's [base fee per gas](../Concepts/Transactions/Transaction-Types.md#eip1559-transactions); the baseFee is burned. Before [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md), this field must empty. After EIP-1559, this field must be set to the block's base fee per gas value. |
```

Comment:
Instead of, "Before EIP-1559, this field must empty. After EIP-1559, this field must be set to the block's base fee per gas value."

Could we just say, "This field is empty for blocks created before EIP-1559"